test/components/vote_counter_test.js

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { shallow } from "enzyme"
+import { spy } from "sinon"
+
+import VoteCounter from "../../web/static/js/components/vote_counter"
+
+describe("VoteCounter", () => {
+  const idea = {
+    category: "sad",
+    body: "redundant tests",
+    user_id: 1,
+    user: {
+      given_name: "Phil",
+    },
+    vote_count: 0,
+  }
+
+  it("renders a button that says Vote", () => {
+    const voteCounter = shallow(
+      <VoteCounter
+        retroChannel={{}}
+        idea={idea}
+      />
+    )
+    const button = voteCounter.find("button")
+
+    expect(button).to.have.length(1)
+    expect(button.text()).to.equal("Vote")
+  })
+
+  it("renders a pointing label that contains the vote count of the idea", () => {
+    const voteCounter = shallow(
+      <VoteCounter
+        retroChannel={{}}
+        idea={idea}
+      />
+    )
+    const label = voteCounter.find("a")
+
+    expect(label).to.have.length(1)
+    expect(label.hasClass("pointing label")).to.be.true
```

Comment:
let's just have one expectation here, that the test of the label matches the idea's current vote count. and no need to specify that it's a pointing label